plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/gitlabMergeRequest.ts

```diff
@@ -57,7 +57,7 @@ export const createPublishGitlabMergeRequestAction = (options: {
           repoUrl: {
             type: 'string',
             title: 'Repository Location',
-            description: `Accepts the format 'gitlab.com/group_name/project_name' where 'project_name' is the repository name and 'group_name' is a group or username`,
+            description: `Accepts the format 'gitlab.com?repo=project_name&owner=group_name' where 'project_name' is the repository name and 'group_name' is a group or username`,
```

Comment:
There's also the possibility of defining a `project` query in some form too as far as I can tell, but I'm feeling this is enough of a step in the right direction that we should ship this as is and can leave that for potential future improvements

Reply:
I guess that such a URL format is a bad idea. But for now, I just adjusted the description and current implementation. It was quite tricky to debug why format that described in description not works